### DISCUSSION TITLE
⚠️ ⚠️ ⚠️ Getting asked for a passphrase for a private key, but there is none

### DISCUSSION THREAD

**Discussion by remino** [isontheline/pro.webssh.net#1156] [Q&A] [answered]
Hi,

Starting today, none of my SSH keys in WebSSH key work. No matter which key or host I try to connect to, I always get the same error:

> Connection error
Unable to parse the private key

In the logs of WebSSH, trying to open any key yields the same error every time:

```
Private Key ([redacted]) need to be upgraded

<EngineLog>
  <FromOpenSshPrivateKey>
    <EngineVersion>9.5.0.98</EngineVersion>
    <EngineStatus>0</EngineStatus>
    <error>The previous call to UnlockBundle failed. Check the LastErrorText for the previous call to UnlockBundle.</error>
  </FromOpenSshPrivateKey>
</EngineLog>
```

Note the SSH keys have not changed neither on the client side or server side, and with other apps, they still work just fine.

I also tried deleting my keys in WebSSH and re-add them, but that had no effect. I also tried changing my key to one that is not password-protected: that prompted me for a password when trying to connect but ended up showing the same error message than above.

Can that be fixed?

- OS: iOS 17.5.1
- Device: iPhone 14 Pro Max
- WebSSH: 26.7.1370
- Key type: ed25519
- Key boundary: `-----BEGIN OPENSSH PRIVATE KEY-----`
- Key is password protected: Yes

**Comment by isontheline** [comment, marked as answer]
@remino Please follow #1153
